r2/r2/lib/jsontemplates.py

```diff
@@ -501,6 +501,14 @@ def thing_attr(self, thing, attr):
         else:
             return SubredditJsonTemplate.thing_attr(self, thing, attr)
 
+class SubredditSidebarJsonTemplate(SubredditJsonTemplate):
+    _data_attrs_ = dict(
+        usertext_md="description",
+        usertext_html="description_html",
```

Comment:
@xiongchiamiov I was originally going to stick with `description` and `description_html` but then I changed to what is there now based off of the css classes that were on the site if you'd inspect element and check the sidebar or /about/sidebar. I don't really care what the name is and I can change it back I'd you'd like.